schemas/app-interface/template-collection-1.yml

```diff
@@ -22,6 +22,14 @@ properties:
     type: string
   enableAutoApproval:
     type: boolean
+  forEach:
+    type: object
+    properties:
+      items:
+        type: array
+        description: object items on which to run this collection
+        items:
+          type: object
```

Comment:
Any other properties under `forEach`?

How about directly make `forEach` as list of object, like 

```yaml
forEach:
- cluster_name: app-sre-prod-01
  value: value1
- cluster_name: app-sre-prod-04
  value: value2
```

Reply:
next step would be to add an alternative to `items` based on `query` + `jsonpath`

Reply:
oh, I see, another layer of static and dynamic, right?

Reply:
it would be either one or the other
```yaml
forEach:
  items: [...]
```
or something like
```yaml
forEach:
  query: |...  # gets data from graphql
  jsonpath: ...  # exctracts a list of items from the graphql result
```